src/js/facility-locator/components/SearchControls.jsx

```diff
@@ -19,19 +27,32 @@ class SearchControls extends Component {
   handleSearch = (e) => {
     const { onSearch } = this.props;
     e.preventDefault();
+    this.toggleVisibility();
 
     onSearch();
   }
 
-  render() {
-    const style = {
-      padding: '1.5rem 1rem',
-    };
+  toggleVisibility = () => {
```

Comment:
@bshyong -- Should `toggleVisibility()` run anytime a search happens? I noticed that if I click `allow ` for geolocation my address is put into the search bar and result are rendered on the map but I can't see them because the search box is not hidden yet.